sample_workloads/nccltest/gke/values.yaml

```diff
@@ -0,0 +1,60 @@
+cluster:
+  nNodes: null  # Must specify on commandline (--set cluster.nNodes=2)
+  sbPlacement: false # Set `true` if running across multiple superblocks
+  nSuperblocks: 1
+  startSuperblock: 1
+  gcsBucket: "pirillo-sct-bucket"
+
+ncclBenchmarks:
+  image: "us-central1-docker.pkg.dev/supercomputer-testing/pirillo-gcr/nccltest"
```

Comment:
Default value not accessible externally (TODO: make this an image under us-docker.pkg.dev/gce-ai-infra/...)